x-pack/test/accessibility/apps/management.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects([
+    'common',
+    'settings',
+    'header',
+    'indexPatternFieldEditorObjects',
+    'indexManagement',
+  ]);
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const a11y = getService('a11y');
+
+  describe('Management', () => {
+    it('main view', async () => {
+      await a11y.testAppSnapshot();
+    });
+
+    describe('index management', async () => {
+      describe('indices', async () => {
+        it('empty state', async () => {
+          await PageObjects.settings.navigateTo();
+          await PageObjects.settings.clickIndexManagement();
+          await a11y.testAppSnapshot();
+        });
+        describe('indices with data', async () => {
+          before(async () => {
```

Comment:
Nit pick : Can the before() and after() method be abstracted above describe() - no harm either way

Reply:
We need these before() and after() in this describe because we want to add the data `after` the first test where there is no data being used.